documentation/modules/upgrading/con-upgrade-strategies-for-upgrading-clients.adoc

```diff
@@ -24,39 +24,45 @@ Broker down-conversion is configured in two ways:
 
 Messages published to a topic in a new-version format will be visible to consumers, because brokers perform down-conversion when they receive messages from producers, not when they are sent to consumers.
 
-There are a number of strategies you can use to upgrade your clients:
+Common strategies you can use to upgrade your clients are described as follows.
+Other strategies for upgrading client applications are also possible.
+
+IMPORTANT: The steps outlined in each strategy change slightly when upgrading to Kafka {DefaultKafkaVersion} or later.
+From Kafka {DefaultKafkaVersion}, the message format version values are assumed to match the `inter.broker.protocol.version` and don't need to be set.
```

Comment:
As the `{DefaultKafkaVersion}` will change, I think you should hardcode 3.0.0 here.